webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -20,31 +20,21 @@ class ManageManifestModal extends Component {
     super(props);
 
     this.state = {
-      actionInProgress: props.taskInProgress,
+      redhat_repository_url: null,
+      hideModalAfterAction: true,
     };
   }
 
-  static getDerivedStateFromProps(newProps, prevState) {
-    if (
-      !isEqual(newProps.taskInProgress, prevState.actionInProgress)
-    ) {
-      return {
-        actionInProgress: newProps.taskInProgress,
-      };
+  componentWillMount() {
```

Comment:
The React folks _really_ don't like the `componentWillMount` method and in general strongly discourage it.  We'll have to start calling it `UNSAFE_componentWillMount` once we upgrade to React v17.  However, I can see why it makes sense here.  Our use case is rather unique and I don't think it breaks any of their [guidelines](https://reactjs.org/docs/react-component.html#unsafe_componentwillmount), so I'm okay with it.